music21/romanText/writeRoman.py

```diff
@@ -303,53 +330,6 @@ def getChordString(self,
 
 # ------------------------------------------------------------------------------
 
-def rnString(measureNumber: int,
-             beat: t.Union[str, int, float, fractions.Fraction],
-             chordString: str,
-             inString: t.Optional[str] = ''):
-    '''
-    Creates or extends a string of RomanText such that the output corresponds to a single
-    measure line.
-
-    If the inString is not given, None, or an empty string then this function starts a new line.
-
-    >>> lineStarter = romanText.writeRoman.rnString(14, 1, 'G: I')
-    >>> lineStarter
-    'm14 G: I'
-
-    For any other inString, that string is the start of a measure line continued by the new values
-
-    >>> continuation = romanText.writeRoman.rnString(14, 2, 'viio6', 'm14 G: I')
-    >>> continuation
-    'm14 G: I b2 viio6'
-
```

Comment:
removal of well-explained, stable, and tested code for undocumented and untested routines.  negative.